### DISCUSSION TITLE
Could not read config file

### DISCUSSION THREAD

**Discussion by iamayushag** [projectdiscovery/subfinder#813] [Q&A]

**Comment by ehsandeep** [comment]
@technicaljunkie you are running outdated version of the project, make sure to update to latest one.

**Reply by ehsandeep** [reply]
You are not running the latest version, you can confirm it with `subfinder -version` option, latest one is `v2.5.7`

You can also use pdtm to update/install - https://github.com/projectdiscovery/pdtm

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
